scripts/create-clusters.sh

```diff
@@ -77,6 +78,14 @@ function configure-insecure-registry-and-reload() {
   ${cmd_context} "$(reload-docker-daemon-cmd "${docker_pid}")"
 }
 
+function configure-insecure-registry-and-reload-for-containerd() {
+  local cmd_context="${1}" # context to run command e.g. sudo, docker exec
+  local containerd_pid="${2}"
+  ${cmd_context} "sed -i 's/\"docker.io\"/\"${CONTAINER_REGISTRY_HOST}\"/' ${containerd_config}"
+  ${cmd_context} "sed -i 's/https:\/\/registry-1.docker.io/http:\/\/${CONTAINER_REGISTRY_HOST}/' ${containerd_config}"
+  ${cmd_context} "$(reload-docker-daemon-cmd "${containerd_pid}")"
```

Comment:
Consider renaming `reload-docker-daemon-cmd` to just `reload-daemon-cmd` since it's used for both `dockerd` and `containerd` now.